.travis.yml

```diff
@@ -14,4 +14,4 @@ script:
 - make test lint
 after_success:
 - grunt bench
-- grunt sauce
+#- grunt sauce
```

Comment:
Let's delete this line. We can always consult the file's history to see earlier versions.